### DISCUSSION TITLE
Bug Daily Puzzle 17/12/2023 ?

### DISCUSSION THREAD

**Discussion by creativeSpecies** [cssbattle/forum#374] [General]
![image](https://github.com/cssbattle/forum/assets/17890885/bfc323a0-5b1f-4b64-a9b0-559cc3d5c4f8)

I don't understand why it states 99% match, there is no visible diff.

Here is the code :
<p><style>*{color:#E5EDE9;background:#004928;border:27q solid;margin:75;>*{margin:-25 150-25-25}p{margin:25 0;border-bottom:0

**Comment by lmicra** [comment]
The match is broken, check the match of the top solutions for [2024-02-13](https://cssbattle.dev/play/5tYZeO2j9PK3vJPJhYOT).

They do not provide 100% match in my PC: [video](https://streamable.com/u5x7o6)

**Comment by lmicra** [comment]
Am I the only one that goes through the top solutions and find them to usually not match 100%. It happen again with 5/3/2024 target, all the top solutions fail!

**Reply by jrojko** [reply]
From FAQs:
Important points to Note:
It is recommended to play on Chromium based browsers (Chrome/Safari) because that is what is used for scoring at the backend. **Other browser may have some differences in how they render a particular code.**

**Comment by eliot146** [comment]
all top solutions are failed
